src/sentry/static/sentry/app/components/platformList.tsx

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import styled from '@emotion/styled';
+
+import PlatformIcon from 'app/components/platformIcon';
+import {PlatformKey} from 'app/data/platformCategories';
+
+type Props = {
+  className?: string;
```

Comment:
Total NIT: Can we move className to the bottom, so that Props are typed in the same order as are later destructured? 😅  It's imho easier to read and scan that way

Reply:
sure :)